Week-5/exercise5.4/calculator.test.js

```diff
@@ -0,0 +1,12 @@
+const {sum,diff,product}=require("./calculator");
+
+test("Returns the sum of the two numbers",()=>{
```

Comment:
[For your learning] Add some negative test cases as well.. for ex : if you pass string to add method as in sum('1','2') result will be 12 right.. so add a functionality in the code such that data type is only numbers.. if the parameters are string return "number are not integers".. add this test case as well..